scripts/pre_commit_linter.py

```diff
@@ -463,8 +470,235 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
+def _get_all_files():
+    """This function is used to check if this script is ran from
+    root directory and to return a list of all the files for linting and
+    pattern checks.
+    """
+    eslintignore_path = os.path.join(os.getcwd(), '.eslintignore')
+    parsed_args = _PARSER.parse_args()
+    # Default mode is non-verbose mode.
+    mode = 'nv'
```

Comment:
If the mode is boolean, represent it as a boolean rather than as a string.

Reply:
Done